whoisit/__init__.py

```diff
@@ -1,25 +1,22 @@
-import requests
-from .bootstrap import Bootstrap
-from .query import QueryBuilder, Query
+from .bootstrap import _BootstrapMainModule
 from .parser import parse
-from .logger import get_logger
-from .utils import create_session, get_session, recursive_merge
-from .version import version
```

Comment:
This wasn't an unused import, it was just so that

```python
from whoisit import version
```

Worked, rather than having to do

```python
from whoisit.version import version
```

Reply:
Ahh gotcha. That was my auto format I think. I disabled the format code with ruff but I must have missed the sort imports. Will fix.